### PR TITLE
Add comments and more features to v3 sample chat UI

### PR DESCRIPTION
**Pull request by cardoso**
This is mainly an effort to start organizing and commenting the sample code itself, but also added some features.

1. Added comments (still evolving in style and usefulness, but it's a start)
2. Separated some of the UI logic to highlight the Stream Chat code
3. Moved code which interacts with `ChannelController` to the top and moved most UI code to the bottom.
4. Added member count and online count to top of chat screen
5. Added typing indicator to top of chat screen
6. Added delete channel option in menu (it's crashing for now, same as in channel list screen)

### CODE REVIEW

File: Sample_v3/Samples/SimpleChat/SimpleChatViewController.swift [hunk 1]
```diff
@@ -5,77 +5,194 @@
 import StreamChatClient
 import UIKit
 
-class SimpleChatViewController: UITableViewController {
-    private var controller: ChannelController?
+///
+/// # SimpleChatViewController
+///
+/// A `UITableViewController` superclass that displays and manages a channel.  It uses the `ChannelController`  class to make calls to the Stream Chat API and listens to
```
Comment (VojtaStavik) [inline Sample_v3/Samples/SimpleChat/SimpleChatViewController.swift:11]: A `UITableViewController` ~superclass~ subclass ...

**Review by VojtaStavik — APPROVED** [review]
Good idea with the comments a changing the code organization 👍 

The crashing you mention - is it because of the SDK or is it intentional? I don't see anything related to crashing on purpose in the PR.

**Comment by cardoso** [comment]
@VojtaStavik deletion is crashing both in this screen and the channels list screen. Not related to this PR

File: Sample_v3/Samples/SimpleChat/SimpleChatViewController.swift (hunk 1 above)
Comment (cardoso) [reply Sample_v3/Samples/SimpleChat/SimpleChatViewController.swift]: Oops :)

**Pull request merged**